.gitignore

```diff
@@ -5,4 +5,5 @@ node_modules
 lib
 package-lock.json
 .idea
-/test-results
\ No newline at end of file
+/test-results
+pnpm-workspace.yaml
```

Comment:
I recommend you just checkout any unexpected files like this one. Don't add them to `.gitignore` (nothing wrong with committing a workspace config one day).

Reply:
sounds good